corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -372,6 +372,75 @@ def test_prompt_daterange(self, *args):
         """
         self.assertXmlPartialEqual(expected, suite, "./remote-request[1]/session/query/prompt[@key='name']")
 
+    def test_prompt_address(self, *args):
+        """Setting the appearance to "address"
+        """
+        # Shouldn't be included for versions before 2.50
```

Comment:
Oh yes, we could do that. I added this before that discussion. 

Does this mean, we would need to explicitly provide config options for address/receiver regardless of app version in [the config screen](https://github.com/dimagi/commcare-hq/pull/29209/files#diff-a669f5780a028b3d02375c2c83fa285a5f2d58789a68a43e03bc1009a0cd94e6R68) as well?

Reply:
I think so, yes. You could use [this check](https://github.com/dimagi/commcare-hq/blob/9f9be0656cafef088da91835e92c061d3e9b586d/corehq/apps/app_manager/views/modules.py#L133), since that'll limit it to web apps, and all of the recent USH case claim changes are going to be moved under that autolaunch flag.

Reply:
Actually, the Format dropdown option is available only for apps > 2.50. For apps < 2.50, should we display a dropdown with a single option of 'Geocoder' (or a checkbox)?

Reply:
I don't have a strong opinion - since this is web apps only, CC versions don't mean much so I don't think it's a big deal to force people to upgrade to 2.50 to get the geocoder. (Original comment was just asking for clarification.)